New formula: mysqltuner

arnested opened the pull request:
Reviving #7365 from @bessl.

mikemcquaid commented on Library/Formula/mysqltuner.rb:
```diff
@@ -0,0 +1,13 @@
+class Mysqltuner < Formula
+  url "https://github.com/major/MySQLTuner-perl/archive/v1.4.0.tar.gz"
+  sha256 "8d5a03b64da2164c6bd93b79700c93db088a14155bcc8cb63c65d049909d793e"
+  homepage "http://mysqltuner.com"
```
Put `homepage` above `url`, thanks!

denji replied:
head "https://github.com/major/MySQLTuner-perl.git"

mikemcquaid commented on Library/Formula/mysqltuner.rb:
```diff
@@ -0,0 +1,13 @@
+  homepage "http://mysqltuner.com"
+
+  def install
+    bin.install "mysqltuner.pl" => "mysqltuner"
```
Are there any other tools that are worth installing?

mikemcquaid commented on Library/Formula/mysqltuner.rb:
```diff
@@ -0,0 +1,13 @@
+  end
+
+  test do
+    system "mysqltuner", "--help"
```
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

arnested commented:
- Moved `homepage` above `url`
- Added `head`
- mysqltuner analyzes your database configuration by connecting to a mysql server. It is not really feasible to spawn a mysql server just for a test case so we'll stick with a rudimentary test. I added a comment about.

mikemcquaid commented:
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `mysqltuner 1.4.0 (new formula)`.
